templates/notices/error.php

```diff
@@ -10,26 +10,24 @@
  * happen. When this occurs the version of the template file will be bumped and
  * the readme will list any important changes.
  *
- * @see         https://docs.woocommerce.com/document/template-structure/
- * @package     WooCommerce/Templates
- * @version     3.5.0
+ * @see     https://docs.woocommerce.com/document/template-structure/
+ * @package WooCommerce/Templates
+ * @version 3.9.0
  */
 
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
 
-if ( ! $messages ) {
+if ( ! $notices ) {
 	return;
 }
 
 ?>
 <ul class="woocommerce-error" role="alert">
-	<?php foreach ( $messages as $message ) : ?>
-		<li>
-			<?php
-				echo wc_kses_notice( $message );
-			?>
+	<?php foreach ( $notices as $notice ) : ?>
+		<li<?php echo wc_get_notice_data_attr( $notice ); ?>>
```

Comment:
This is not a blocker, but I believe we need to add `wc_get_notice_data_attr()` to `WordPress.Security.EscapeOutput` in the `woocommerce-sniffs` repository?

Reply:
Yes, that's my idea, to include on WooCommerce Sniffs after we merge it.